python/snewpy/scripts/SNEWS2.0_rate_table.py

```diff
@@ -1,190 +1,193 @@
+# For IceCube & KM3NeT, the effective masses in SNOwGLoBES are artificially high.  This is because the
+# non-standard energy dependence is handled through the efficiencies.  To get an effective
+# mass we take the ratio of the total weighted events to the unweighted events and multiply
+# the unweighted mass (the entry in SNOwGLoBES), see below for details.  Here we take the
+# effective mass of the s27 normal scenario and discuss the range in the table caption.
 
 dettype='icecube'
 mass=51600
 data['Mass [kt]'][2] = "~"+str(100*int(0.01*(mass*total_events['s27.0']['AdiabaticMSW_NMO'][dettype+"smeared"]/
       total_events['s27.0']['AdiabaticMSW_NMO'][dettype+"unsmeared"]+50)))+"*"
-
-#DarkSide-20k & Ar/any $\nu$ & 0.02 & Italy & \\
-#For DarkSide-20k, the text on the SNEWS2.0 paper says the s27 will produce 250 events at 10kpc with a
-#detector mass of 47 tonnes
-s27_epT = 5319.1489361703
-
-mass = 0.047
-data['Experiment'].append('DarkSide-20k')
-data['Type'].append('Ar/any $\\nu$')
-data['Mass [kt]'].append(mass)
-data['Location'].append('Italy')
-data['\\SI{11.2}{\solarmass}'].append("-")
-data['\\SI{27.0}{\solarmass}'].append(int(mass*s27_epT))
-data['\\SI{40.0}{\solarmass}'].append("-")
-
-
-#For XENONnT, LZ, and PandaX-4T, we take the values from Lang et al. [Phys. Rev. D 94 (2016) no.10, 103009].
-#We take the S2-only, 60PE threshold numbers for the 11.2 msun and 27 msun models as they
-#are the ones used here. We take 9.4 events/tonne for the 11.2 model and 17.6 events/tonne for the 27.0 model.
-s11_epT = 9400.
-s27_epT = 17600.
-
-mass = 0.008
-data['Experiment'].append('XENONnT')
-data['Type'].append('Xe/any $\\nu$')
-data['Mass [kt]'].append(mass)
-data['Location'].append('Italy')
-data['\\SI{11.2}{\solarmass}'].append(int(mass*s11_epT))
-data['\\SI{27.0}{\solarmass}'].append(int(mass*s27_epT))
-data['\\SI{40.0}{\solarmass}'].append("-")
-
-mass = 0.007
-data['Experiment'].append('LZ')
-data['Type'].append('Xe/any $\\nu$')
-data['Mass [kt]'].append(mass)
-data['Location'].append('USA')
-data['\\SI{11.2}{\solarmass}'].append(int(mass*s11_epT))
-data['\\SI{27.0}{\solarmass}'].append(int(mass*s27_epT))
-data['\\SI{40.0}{\solarmass}'].append("-")
-
-mass = 0.004
-data['Experiment'].append('PandaX-4T')
-data['Type'].append('Xe/any $\\nu$')
-data['Mass [kt]'].append(mass)
-data['Location'].append('China')
-data['\\SI{11.2}{\solarmass}'].append(int(mass*s11_epT))
-data['\\SI{27.0}{\solarmass}'].append(int(mass*s27_epT))
-data['\\SI{40.0}{\solarmass}'].append("-")
-
-
-ascii.write(data,Writer=ascii.Latex)
+# +50 in line 143 is for rounding so we have a nice number for the table
+
+dettype='km3net'
+mass=69366 * 3
+data['Mass [kt]'][3] = "~"+str(100*int(0.01*(mass*total_events['s27.0']['AdiabaticMSW_NMO'][dettype+"smeared"]/
+      total_events['s27.0']['AdiabaticMSW_NMO'][dettype+"unsmeared"]))+50)+"*"
+# +50 in line 145 to get 150 as an output, the average between normal (100) & inverted (200)
```

Comment:
Isn’t this the same rounding as for `icecube` a few lines above?

Reply:
We are not sure why it was there for IceCube, and for IceCube, it is there only inside the denominator (unsmeared), not out of the parenthesis...